Dockerfile

```diff
@@ -18,3 +18,5 @@ COPY --chown=node:node server-config*.json ./
 COPY --chown=node:node dist ./dist/
 COPY --chown=node:node scripts ./scripts/
 EXPOSE 8090
+
+CMD ["node", "dist/commands/Start.js", "--config", "server-config.json"]
```

Comment:
What's the purpose of defining the command here instead of defining it in the docker-compose? and why are we using `CMD` instead of `ENTRYPOINT`?

Reply:
The purpose is to make the container independent of Docker compose or any entity in charge of sending an entry point. In the other way, without an external entity (docker compose in this case) sending an entry point, the container is useless